dev/NavigationView/NavigationView_ApiTests/NavigationViewTests.cs

```diff
@@ -572,5 +573,82 @@ private void VerifyVerifyHeaderContentMargin(NavigationViewPaneDisplayMode paneD
                 root: headerContent,
                 masterFilePrefix: masterFilePrefix);
         }
+
+        [TestMethod]
+        public void VerifyMenuItemAndContainerMappingMenuItemsSource()
+        {
+            NavigationView navView = null;
+            ObservableCollection<String> menuItems = new ObservableCollection<String>();
+            menuItems.Add("Item 1");
+            menuItems.Add("Item 2");
+
+            RunOnUIThread.Execute(() =>
+            {
+                navView = new NavigationView();
+                MUXControlsTestApp.App.TestContentRoot = navView;
+
+                navView.MenuItemsSource = menuItems;
+                navView.Width = 1008; // forces the control into Expanded mode so that the menu renders
+
+                MUXControlsTestApp.App.TestContentRoot.UpdateLayout();
+
+                var menuItem = "Item 2";
+                // Get container for item
+                var itemContainer = navView.ContainerFromMenuItem(menuItem) as NavigationViewItem;
+                bool correctContainerReturned = itemContainer != null && (itemContainer.Content as String) == menuItem;
+                Verify.IsTrue(correctContainerReturned, "Correct container should be returned for passed in menu item.");
+
+                // Get item for container
+                var returnedItem = navView.MenuItemFromContainer(itemContainer) as String;
+                bool correctItemReturned = returnedItem != null && returnedItem == menuItem;
+                Verify.IsTrue(correctItemReturned, "Correct item should be returned for passed in container.");
+
+                MUXControlsTestApp.App.TestContentRoot = null;
```

Comment:
I dont see all tests in this class doing this. This might make sense to do in the test cleanup method.

Reply:
Make sense. For now it doesn't look to be making a difference since other tests overwrite the content root before operating. For this PR I just want to get these test in, however i've made note of looking into making this update in another set of changes.